cves/kernel/CVE-2021-3764.yml

```diff
@@ -55,7 +55,9 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  This vulnerability is a memory leak in the kernel's ccp_run_aes_gcm_cmd() function. 
```

Comment:
Description can be more fluid paragraph that goes together instead of standalone sentences